lib/travis/build/addons/pkg.rb

```diff
@@ -0,0 +1,69 @@
+require 'travis/build/addons/base'
+require 'shellwords'
+
+module Travis
+  module Build
+    class Addons
+      class Pkg < Base
+        SUPPORTED_OPERATING_SYSTEMS = %w[
+          freebsd
+        ].freeze
+
+        def before_prepare?
+          SUPPORTED_OPERATING_SYSTEMS.any? do |os_match|
+            data[:config][:os].to_s == os_match
+          end
+        end
+
+        def before_prepare
+          return if config_pkg.empty?
+          sh.newline
+          sh.fold('pkg') do
+            install_pkg
+          end
+          sh.newline
+        end
+
+        def before_configure?
+          # keeping empty for now
+        end
+
+        def before_configure
+        sh.echo "Configuring default pkg options", ansi: :yellow
```

Comment:
Perhaps indent the contents of this function one level?

Would it be possible to add a test for this bit? And perhaps to try this out in staging to make sure what I suggested here actually works properly? 😬 cc also @lwhsu, as the branch handling bit here was adapted from [bento](https://github.com/lwhsu/bento).

Reply:
What I can say for now is the code and the path in this part look good to me. :)